pkg/store/postgres_backend/project_store.go

```diff
@@ -0,0 +1,81 @@
+package postgres_backend
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/discmonkey/retext/pkg/store"
+	"time"
+)
+
+type ProjectStore struct {
+	con *sql.DB
+}
+
+func NewProjectStore() (ProjectStore, error) {
+	con, err := GetConnection()
```

Comment:
`GetConnection` creates a new poll of DB connections each time it's called (via `sql.Open()`.  That's not recommended; [`sql.Open` should be called only once](https://pkg.go.dev/database/sql#Open).